infrastructure/lambda/template.yaml

```diff
@@ -200,8 +200,6 @@ Resources:
       Name: !Sub ${AWS::StackName}-PublicUKPassportApi
       Description: Public UK Passport CRI API
       StageName: !Ref Environment
-      Auth:
```

Comment:
Will this not also remove the protection in the environments where its needed?

Reply:
That is the reason I had left in, I been able to confirm that the public api gateway methods all require an api key when in a pipeline deployment configuration. Passport now matches fraud and dl for how this operates.